nexus/test-utils/src/background.rs

```diff
@@ -8,27 +8,27 @@ use crate::http_testing::NexusRequest;
 use dropshot::test_util::ClientTestContext;
 use nexus_client::types::BackgroundTask;
 use nexus_client::types::CurrentStatus;
-use nexus_client::types::CurrentStatusRunning;
 use nexus_client::types::LastResult;
 use nexus_client::types::LastResultCompleted;
 use nexus_types::internal_api::background::*;
 use omicron_test_utils::dev::poll::{wait_for_condition, CondCheckError};
 use std::time::Duration;
 
-/// Return the most recent start time for a background task
-fn most_recent_start_time(
+/// Return the most recent activate time for a background task, returning None
```

Comment:
Is my understanding correct that this is returning:  
```
The activate time for the last completed background task.
Returning None if the task is currently running, or has never run.
```
You only get a `Some` here if the task has completed at least once and is not currently running?

Reply:
Correct, yeah.